views/welcome.html

```diff
@@ -83,8 +84,22 @@ <h2>Integrations</h2>
           </div>
         </div>
       </div>
-      
-      
+
+
+      <div class="tab-pane fade show" id="list-create-azure-host-policy" role="tabpanel" aria-labelledby="list-create-azure-host-policy-list">
+        <div class="row">
+          <div class="col-6">
+            <Br/>
+            <h2>Create Azure Host Policy</h2>
+            Generate the Host policy needed to authenticate using Azure Authenticator from Azure token<br/>
+
+            <button type="button" class="btn btn-primary gotolink" data-page="/guides/policy/azure-authenticator">Continue</button>
+
+          </div>
+        </div>
+      </div>
+
```

Comment:
extra few lines here

Reply:
This is how @quincycheng divided between the `div`s in the welcome page.